src/amo/components/UserProfile/index.js

```diff
@@ -0,0 +1,170 @@
+      }));
+    }
+  }
+
+  render() {
+    const {
+      errorHandler,
+      i18n,
+      params,
+      user,
+    } = this.props;
+
+    if (errorHandler.hasError()) {
+      log.warn('Captured API Error:', errorHandler.capturedError);
+
+      if (errorHandler.capturedError.responseStatusCode === 404) {
+        return <NotFound errorCode={errorHandler.capturedError.code} />;
+      }
+    }
+
+    const userProfileHeader = (
+      <React.Fragment>
+        <UserAvatar className="UserProfile-avatar" user={user} />
+
+        <h1 className="UserProfile-name">
+          {user ? user.displayName : <LoadingText />}
+        </h1>
+      </React.Fragment>
+    );
+    const userProfileTitle = i18n.sprintf(
+      i18n.gettext('User Profile for %(user)s'), {
+        user: user ? user.displayName : params.username,
+      }
+    );
+
+    return (
+      <div className="UserProfile">
+        <Helmet>
+          <title>{userProfileTitle}</title>
+        </Helmet>
+
+        {errorHandler.renderErrorIfPresent()}
+
+        <div className="UserProfile-wrapper">
+          <Card
+            className="UserProfile-user-info"
+            header={userProfileHeader}
+          >
+            <DefinitionList className="UserProfile-dl">
+              {user && user.homepage ? (
+                <Definition
+                  className="UserProfile-homepage"
+                  term={i18n.gettext('Homepage')}
+                >
+                  {/* TODO: Make a helper that strips https?:\/\/ from text */}
+                  <a href={user.homepage}>{user.homepage.replace(/^https?:\/\//, '')}</a>
+                </Definition>
+              ) : null}
+              <Definition
+                className="UserProfile-user-since"
+                term={i18n.gettext('User since')}
+              >
+                {user ? i18n.moment(user.created).format('ll')
+                      : <LoadingText />}
+              </Definition>
+              <Definition
+                className="UserProfile-number-of-addons"
+                term={i18n.gettext('Number of add-ons')}
+              >
+                {user ? user.num_addons_listed : <LoadingText />}
+              </Definition>
+              <Definition
+                className="UserProfile-rating-average"
+                term={i18n.gettext('Average rating of developer’s add-ons')}
+              >
+                {user ? (
+                  <Rating
+                    rating={parseFloat(user.average_addon_rating)}
+                    readOnly
+                    styleName="small"
+                  />
+                ) : <LoadingText />}
+              </Definition>
+            </DefinitionList>
+
+            {user && user.biography && user.biography.length ? (
+              <div
+                className="UserProfile-biography"
+                // eslint-disable-next-line react/no-danger
+                dangerouslySetInnerHTML={sanitizeUserHTML(user.biography)}
```

Comment:
The old site supported a bunch of tags. We should probably start by doing the same:

<img width="860" alt="screenshot 2018-03-14 11 59 49" src="https://user-images.githubusercontent.com/55398/37417835-42d867e2-277f-11e8-9bea-5cd2e222733b.png">

Reply:
That's exactly what `sanitizeUserHTML` does 😄:

https://github.com/mozilla/addons-frontend/blob/master/src/core/utils/index.js#L90

Reply:
oh duh, I read it as `sanitizeHTML()` for some reason

Reply:
Hah, well that’s easy to mix up 😅